script.js

```diff
@@ -0,0 +1,29 @@
+const cardsContainer = document.querySelector(".cards");
+
+const colors = [
+  "pink",
+  "aqua",
+  "crimson",
+  "blue",
+  "gold",
+  "greenyellow",
+  "teal",
+  "aquamarine",
+  "dodgerblue",
+];
+
+const colorPickList = [...colors, ...colors];
+const color = colorPickList[0];
+
+function buildCard(color) {
+  const element = document.createElement("div");
+  element.classList.add("card");
+  element.setAttribute("background-color", color);
+  element.addEventListener("click", () => {
+    element.style.backgroundColor = color;
+  });
+  return element;
+}
+
+const card = buildCard(color);
+cardsContainer.appendChild(card);
```

Comment:
I am wondering a bit what is the idea behind having all these colors here and using one specific color deliberately?  
Perhaps this is temporary and you will do something else with the pictures later 🤔